server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyRequestBody.scala

```diff
@@ -0,0 +1,32 @@
+package sttp.tapir.server.netty
+
+import scala.concurrent.Future
+
+import io.netty.handler.codec.http.{FullHttpMessage, FullHttpRequest, HttpContent}
+import sttp.capabilities
+import sttp.tapir.RawBodyType
+import sttp.tapir.server.interpreter.{RawValue, RequestBody}
+import scala.concurrent.ExecutionContext.Implicits.global
+
+import io.netty.buffer.ByteBufInputStream
+import sttp.tapir.internal.NoStreams
+
+//todo: ChannelFuture or something with cats-effect?
+//todo: FullHttpMessage OK?
+class NettyRequestBody(req: FullHttpMessage) extends RequestBody[Future, NoStreams] {
+
+  override val streams: capabilities.Streams[NoStreams] = NoStreams
+
+  override def toRaw[RAW](bodyType: RawBodyType[RAW]): Future[RawValue[RAW]] = {
+    bodyType match {
+      case RawBodyType.StringBody(charset) => Future(RawValue(req.content().toString(charset)))
```

Comment:
since these are strict values, you should use `Future.successful` instead. `Future.apply` will send this computation to another thread without any need